### PR TITLE
Fix the colors on the colored social buttons

### PR DESCRIPTION
**Pull request by dshukertjr**
## What kind of change does this PR introduce?

There has been a report that certain configuration on the theme causes the design of the colored social buttons to have some problems. This PR fixes some of the color values on the colored social buttons so that the design of the colored social buttons would not get overridden by the theme settings.

fixes https://github.com/supabase-community/flutter-auth-ui/issues/41
fixes https://github.com/supabase-community/flutter-auth-ui/issues/40

### CODE REVIEW

File: lib/src/components/supa_socials_auth.dart [hunk 1]
```diff
@@ -153,62 +153,43 @@ class _SupaSocialsAuthState extends State<SupaSocialsAuth> {
...
+            iconWidget = Image.asset(
+              'assets/logos/google_light.png',
+              package: 'supabase_auth_ui',
+              width: 48,
+              height: 48,
             );
+
+            foregroundColor = Colors.black;
+            backgroundColor = Colors.white;
+            overlayColor = Colors.white;
```
Comment (dshukertjr) [inline lib/src/components/supa_socials_auth.dart:180]: Only for Google login button use a google logo image for icon and black for text. Specifying overlayColor prevents the background color of the image from showing up like this:

image

**Comment by dshukertjr** [comment]
@Csierram96

Would love to hear what you think about this PR!

**Comment by Csierram96** [comment]
> @Csierram96
> 
> Would love to hear what you think about this PR!

The implementation looks good to me. I can give it a try before merging to make sure it works fine.

**Comment by Csierram96** [comment]
The text on the apple button is now visible. 
But i still see white background on the google auth button
Screenshot 2023-02-27 at 9 20 40 PM

**Comment by Csierram96** [comment]
As i see there are 2 solutions: 
1. use a png asset without a background. I tried it and with the asset I used it looks a but pixelated, maybe you can find a better google logo. 
2. Set manually the google button background. I suggest you go with this one (i can also help if you want). Google uses a solution similar to this on their flutter fire ui package. 
image
image

**Comment by Csierram96** [comment]
Also as an additional suggestion it would be a bit better to use an svg instead of a png file to ensure it always looks good

**Comment by dshukertjr** [comment]
@Csierram96 

> The text on the apple button is now visible.
But i still see white background on the google auth button

Sounds good. I will use a higher resolution image with transparent background! BTW, how are you altering the background color of the button here?

**Comment by Csierram96** [comment]
I'm still looking into this. I found out that you're actually setting the color manually, but i just found that it shows as a different color because of it being an elevated button. 
- If i change the elevation of the elevated button to 0 it shows the correct color. 
-  If I change the elevated button to a filledButton and set the elevation manually(for ex to 20) it shows the correct color and shows the button with elevation

**Comment by Csierram96** [comment]
I pulled the changes and looks like this. Are you sure it doesnt have a background?
Screenshot 2023-02-27 at 10 52 15 PM
Screenshot 2023-02-27 at 10 51 59 PM

**Comment by Csierram96** [comment]
This is the one I used that looked pixelated:
google_light

**Comment by dshukertjr** [comment]
LOL 😂 Thanks for checking!

**Pull request merged**